### PR TITLE
This is the exported JSON from the grafana dashboard

### LINKED ISSUE #1337: Move the JSON RPC Relay grafana dashboard configs into the JSON RPC Relay repository
### Problem

As a developers adding metrics to the JSON RPC Relay I would like to have a fast iterative development experience.  All development updates should be done in one place or repository.  This should be the JSON RPC Relay.  The grafana dashboard is part of the product and should be maintained with the product. 

### Solution

- This can broken down into tasks or steps
- First move or copy the grafana dashboard to the JSON RPC Relay repository.  We will now have duplication of the dashboard in both the local node and rpc node repositories.  Also add prometheus configuration to the JSON RPC Relay.
- Integrate the local node repository with the JSON RPC Relay repository.  Research (spike) needed here.  Possible solution: https://stackoverflow.com/questions/15844542/git-symlink-reference-to-a-file-in-an-external-repository
- Define a deployment process for the dashboard to the production grafana server.  Perhaps this means the JSON RPC Relay node has its own dedicated grafana server.  

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by rustyShacklefurd**
**Description**:
This PR modifies ... in order to support ...
* Adds grafana dashboard to charts folder

**Related issue(s)**:
Fixes #1337

**Checklist**
- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-json-rpc/dashboards/JSON RPC Relay (Official)-1689822975711.json [hunk 1]
```diff
@@ -0,0 +1,9538 @@
+{
```
Comment (Nana-EC) [inline charts/hedera-json-rpc/dashboards/JSON RPC Relay (Official)-1689822975711.json:1]: Can we rename this file to `hedera-json-rpc-http-server.json`

**Review by Nana-EC — COMMENTED** [review]
LG, can we just rename the file?

File: charts/hedera-json-rpc/dashboards/JSON RPC Relay (Official)-1689822975711.json (hunk 1 above)
Comment (Nana-EC) [reply charts/hedera-json-rpc/dashboards/JSON RPC Relay (Official)-1689822975711.json:1]: After this we'll create and add one for `hedera-json-rpc-ws-server.json`

**Review by AlfredoG87 — APPROVED** [review]
LGTM

**Pull request merged**